modules/moordyn/src/MoorDyn_Point.f90

```diff
@@ -409,13 +411,13 @@ SUBROUTINE Point_RemoveLine(Point, lineID, TopOfLine, rEnd, rdEnd)
                EXIT
             END DO
             
-            found = 1
+            found = .true.
          
          END IF
          
       END DO
 
-      IF (found == 0) THEN   ! detect if line not found TODO: fix this, its wrong. If pointNnattached is oprginally 2, then it will be 1 after one run of the loop and l will also be 1
+      IF (found) THEN   ! detect if line not found TODO: fix this, its wrong. If pointNnattached is oprginally 2, then it will be 1 after one run of the loop and l will also be 1
```

Comment:
wouldn't this need to be `if (.not. found)`?

Reply:
Doh!  whoops... I'll get that fixed.